js-school/homework-4/src/components/changePlayer.js

```diff
@@ -0,0 +1,12 @@
+import '../consts/index'
+
+const changePlayer = () => {
+  current = 0;
+  document.getElementById('current-'+activePlayer).textContent = 0;
+  document.querySelector(`.player-${activePlayer}-panel`).classList.toggle('active');
+  activePlayer = +!activePlayer;
+  diceElement.style.display = 'none';
+  document.querySelector(`.player-${activePlayer}-panel`).classList.toggle('active');
+}
+
+export default changePlayer()
```

Comment:
It's not good to run `changePlayer` while export in your case.
It will be better:
```js
export default changePlayer;
```

And then in your `components/hold.js` you should import this function and run:

```js
import changePlayer from './components/changePlayer'
// ...
changePlayer();
```

This applies to all modules

P.S. read about modules https://learn.javascript.ru/modules